### ISSUE TITLE
Accessibility: Controls and media elements should have labels

### ISSUE THREAD

**Issue by dlo83** [froala/wysiwyg-editor#1283]
##### Expected behavior.
Froala controls should have labels to be ARIA compliant

##### Actual behavior.
The buttons do not have ARIA labels on them

##### Steps to reproduce the problem.

Example: 

`<button type="button" tabindex="-1" class="fr-command fr-btn" data-cmd="bold"><i class="fa fa-bold"></i></button>`

should be:

`<button type="button" tabindex="-1" **aria-label="Bold"** class="fr-command fr-btn" data-cmd="bold"><i class="fa fa-bold"></i></button>`

##### OS.
All
##### Browser.
All
